### DISCUSSION TITLE
Is there a plan/roadmap to support crio?

### DISCUSSION THREAD

**Discussion by rahtr** [chaos-mesh/chaos-mesh#1598] [Q&A] [answered]
CRIO is default container runtime for openshift. Do we have plans to include podman support in near future. thanks

**Comment by yeya24** [comment]
IIUC podman is not a CRI implementation. CRI-O is the CRI used in Openshift and it was supported recently in https://github.com/chaos-mesh/chaos-mesh/pull/1530.

**Reply by rahtr** [reply]
Thanks,  @yeya24 . My bad, I wrote Podman instead of CRIO. I saw this in the install script and thought it was not supported

`    -r  --runtime            Runtime specifies which container runtime to use. Currently we only supports docker and containerd. default value: docker`

![image](https://user-images.githubusercontent.com/31742059/112571068-a4e8fd00-8dbd-11eb-8bcb-e7cc15dfe52b.png).

I am glad CRIO support has been added.

**Comment by YangKeao** [comment, marked as answer]
The installation script doesn't support CRI-O yet. If you want to install Chaos Mesh on kubernetes with CRI-O, you could try to install it in `helm` with `--set chaosDaemon.runtime="crio" --set chaosDaemon.socketPath="/var/run/crio/crio.sock"` (make sure the `socketPath` is correct :smile_cat: )
